src/frontend/components/ArchiveCard.jsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+//icons
+import { MdDeleteOutline, MdOutlineUnarchive } from "react-icons/md";
+//context
+import { useNotes } from "../context";
+
+export function ArchiveCard() {
+  const { state, restoreArchiveNote, deleteArchiveNote } = useNotes();
+  return (
+    <>
+      {state.archiveList.length < 1 ? (
```

Comment:
instead of `state.archiveList.length < 1` we can write

what if the state is null? Please handle edge case scenarios too

state && state.archiveList.length ? true : false